test/e2e/runner/setup.go

```diff
@@ -150,7 +150,7 @@ func MakeGenesis(testnet *e2e.Testnet) (types.GenesisDoc, error) {
 	if testnet.PbtsUpdateHeight == -1 {
 		genesis.ConsensusParams.Feature.PbtsEnableHeight = testnet.PbtsEnableHeight
 	}
-	for valName, power := range *testnet.Manifest.Validators {
+	for valName, power := range testnet.Validators {
```

Comment:
I it correct here to drop one "level" of dereference? From `testnet.Manifest.Validators` to `testnet.Validators`.

Reply:
I think the code is clear. `Testnet` "extends `Manifest` so it inherits its fields.

We are using it like that in the remaining of this method.

Reply:
You are right 👍